skimage/draw/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Drawing primitives that operate on NumPy arrays, e.g., lines, text, etc."""
```

Comment:
```suggestion
"""Drawing primitives, such as lines, circles, text, etc."""
```

Reply:
Current phrasing makes the "e.g" sound like it applies to NumPy arrays, which is confusing.

Reply:
Addressed in https://github.com/scikit-image/scikit-image/pull/6892/commits/f69ba1cbd53342d5c2221c1fc9915e634eca5e8c.